code/__defines/reagent_data_fields.dm

```diff
@@ -0,0 +1,60 @@
+// Currently just used for indexing reagent data without
+// having to use strings. UID is provided for serde.
+
+/decl/reagent_data_field
```

Comment:
not super sure why this is preferable to string defines? if it works it works though

Reply:
string creation happens once and if the string is compiled in its fine, it's only the creation of new strings that causes string tree rebalancing. that said, a set_data proc that does validation would be nice